src/lib/query.ts

```diff
@@ -179,7 +179,25 @@ export class Query extends BaseQuery {
    * @returns {Query}
    */
   containedIn(key: string, value: (string | number | boolean)[]): Query {
-    this._queryParams[key] = value;
+    this._queryParams[key] = { '$in': value };
+    return this;
+  }
+
+  /**
+   * @method NoContainedIn
+   * @memberof Query
+   * @description Returns the raw (JSON) query based on the filters applied on Query object.
+   * @example
+   * import contentstack from '@contentstack/delivery-sdk'
+   *
+   * const stack = contentstack.Stack({ apiKey: "apiKey", deliveryToken: "deliveryToken", environment: "environment" });
+   * const query = stack.contentType("contentTypeUid").Query;
```

Comment:
@harshithad0703  It should be Query(), please update in all the places.

Reply:
made changes